src/Microsoft.DotNet.XHarness.CLI/Commands/Android/AndroidTestCommand.cs

```diff
@@ -117,6 +132,27 @@ protected override Task<ExitCode> InvokeInternal(ILogger logger)
                         {
                             logger.LogInformation($"Test execution summary:{Environment.NewLine}{resultValues[_testRunSummaryVariableName]}");
                         }
+
+                        // Due to the particulars of how instrumentations work, ADB will report a 0 exit code for crashed instrumentations
+                        // We'll change that to a specific value and print a message explaining why.
+                        if (resultValues.ContainsKey(_returnCodeVariableName))
```

Comment:
nit: you could also `TryGetValue` and use it below at :140 and :147

Reply:
IMO that's a personal preference, doesn't really simplify or make more obvious, so I'm going to skip this.